### PR TITLE
Feat/getter_setter

### PR DESCRIPTION
**Pull request by hoangdv2429**
add CRUD for auction, auction period and bid

### CODE REVIEW

File: module/x/auction/keeper/bid.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	bz := store.Get(bidKey)
+	if len(bz) == 0 {
+		return val, false
+	}
+	k.cdc.MustUnmarshal(bz, &val)
+	return val, true
+}
+
+// GetBidByID returns the bid by auction ID and bidder address
+func (k Keeper) GetBidByAddressAndID(ctx sdk.Context, auctionID uint64, bidderAddress string) (val types.Bid, found bool) {
```
[the remark this answers is not in the document]
Comment (hoangdv2429) [reply module/x/auction/keeper/bid.go:22]: this one is also use to update amount for specific bidder and auction, I'll creat another to get highestBid

File: module/x/auction/keeper/auction_period.go [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+	}
+
+	return auctionPeriods
+}
+
+// GetLatestAuctionPeriod returns the latest auction period.
+func (k Keeper) GetLatestAuctionPeriod(ctx sdk.Context) (val types.AuctionPeriod, found bool) {
+	auctionPeriods := k.GetAllAuctionPeriods(ctx)
+	if len(auctionPeriods) == 0 {
+		return val, false
```
[the remark this answers is not in the document]
Comment (hoangdv2429) [reply module/x/auction/keeper/auction_period.go:44]: okayy

File: module/x/auction/keeper/bid.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (sontrinh16) [reply module/x/auction/keeper/bid.go:22]: we gonna do 1 bid each auction only. so update amount will either replace the old bid or update the amount only so no need to query by address and auction id

[the remark this answers is not in the document]
Comment (hoangdv2429) [reply module/x/auction/keeper/bid.go:22]: agree, I'll remove it

**Review by sontrinh16 — APPROVED** [review]
lgtm

**Pull request merged**